pkg/etcd-backup-controller/controller.go

```diff
@@ -0,0 +1,71 @@
+package controller
```

Comment:
dir should be `etcd-backup-operator/controller`

Reply:
do you mean `pkg/etcd-backup-operator/controller`?

Reply:
right

Reply:
done!